### PR TITLE
Get scheduled task display name with filename 

### PR DESCRIPTION
**Pull request by 0x49736b**
If Principals/Principal/DisplayName is empty, use the filename as the display_name.

I noticed that the field "Principals/Principal/DisplayName" is not used a lot, resulting in the Display Name being null for most of the records.

Instead, let's use the filename as Display Name (Like in the Task Scheduler GUI)

### CODE REVIEW

File: dissect/target/plugins/os/windows/task_helpers/tasks_xml.py [hunk 1]
```diff
@@ -101,6 +101,8 @@ def __init__(self, task_element: Element, task_path: TargetPath):
         self.logon_type = self.get_element("Principals/Principal/LogonType")
         self.group_id = self.get_element("Principals/Principal/GroupId")
         self.display_name = self.get_element("Principals/Principal/DisplayName")
+        if self.display_name == None:
+            self.display_name = str(task_path)[str(task_path).rfind("/")+1:]
```
Comment (Schamper) [inline dissect/target/plugins/os/windows/task_helpers/tasks_xml.py:105]:
```suggestion
        self.display_name = self.get_element("Principals/Principal/DisplayName") or task_path.name
```
I believe this achieves the same result.

**Review by Schamper — CHANGES_REQUESTED** [review]

**Comment by 0x49736b** [comment]
@DissectBot agree

**Comment by Schamper** [comment]
Could you adjust the unit tests with the newly filled in field?

**Comment by 0x49736b** [comment]
> Could you adjust the unit tests with the newly filled in field?

@Schamper , done here https://github.com/fox-it/dissect.target/pull/401/commits/e75a335467853d7643c795f62f5ccc5b307611fe

File: tests/test_plugins_os_windows_tasks.py [hunk 2]
```diff
@@ -40,7 +40,7 @@ def assert_xml_task_properties(xml_task):
     assert xml_task.user_id is None
     assert xml_task.logon_type is None
     assert xml_task.group_id == "S-1-5-4"
-    assert xml_task.display_name is None
+    assert xml_task.display_name is "test_xml.xml"
```
Comment (Schamper) [inline tests/test_plugins_os_windows_tasks.py:43]:
```suggestion
    assert xml_task.display_name == "test_xml.xml"
```
Usually you only use a `is` comparison on things like `None` or class types. Any other type of comparison should be done with `==`.

**Review by Schamper — CHANGES_REQUESTED** [review]

**Review by Schamper — APPROVED** [review]

**Pull request merged**